stips/instruments/instrument.py

```diff
@@ -133,50 +150,112 @@ def reset(self, ra, dec, pa, filter, obs_count, psf=True, detectors=True, celery
+#                                                        self.filter,
+#                                                        self.oversample,
+#                                                        self.grid_size,
+#                                                        self.detector)
+#         if os.path.exists(os.path.join(self.out_path, "psf_cache")):
+#             psf_file = os.path.join(self.out_path, "psf_cache", psf_name)
+#             if os.path.exists(psf_file):
+#                 from webbpsf.utils import to_griddedpsfmodel
+#                 if (self.psf_commands is None or self.psf_commands == ''):
+#                     self.psf = to_griddedpsfmodel(psf_file)
+#                     have_psf = True
+#         if not have_psf:
+#             base_state = self.getState()
+#             update_state = "<br /><span class='indented'>Generating PSF</span>"
+#             self.updateState(base_state+update_state)
+#             ins = self.psf_constructor
+#             if self.psf_commands is not None and self.psf_commands != '':
+#                 for attribute,value in self.psf_commands.iteritems():
+#                     setattr(ins,attribute,value)
+#             ins.filter = self.filter
+#             plotplam = self.PHOTPLAM[self.filter]
+#             scale = self.SCALE[0]
+#             safe_size = int(np.floor(30. * photplam / (2. * scale)))
+#             ins_size = max(self.DETECTOR_SIZE) * self.oversample
+#             conv_size = int(np.floor(2048 / self.oversample))
+#             msg = "PSF choosing between {}, {}, and {}"
+#             self._log("info", msg.format(safe_size, ins_size, conv_size))
+#             fov_pix = min(safe_size, ins_size, conv_size)
+#             num_psfs = self.grid_size*self.grid_size
+#             if os.path.exists(os.path.join(self.out_path, "psf_cache")):
+#                 save = True
+#                 overwrite = True
+#                 psf_dir = os.path.join(self.out_path, "psf_cache")
+#                 psf_file = "psf_{}_{}_{}_{}_{}".format(self.INSTRUMENT,
+#                                                        stips_version, 
+#                                                        self.filter,
+#                                                        self.oversample,
+#                                                        self.grid_size)
+#             else:
+#                 save = False
+#                 overwrite = False
+#                 psf_dir = None
+#                 psf_file = None
+#             
+#             self.psf = ins.psf_grid(all_detectors=True, num_psfs=num_psfs,
+#                                     fov_pixels=fov_pix, normalize='last',
+#                                     oversample=self.oversample, save=True,
+#                                     outdir=psf_dir, outfile=psf_file,
+#                                     overwrite=overwrite)
+#             self.updateState(base_state)
```

Comment:
These lines should be removed if they are not being used. If you will need them in the future its best to keep them in a branch in your fork or a local copy on your machine.

Reply:
Changes made and committed.